scripts/dts/gen_defines.py

```diff
@@ -154,8 +154,7 @@ def write_props(dev):
             continue
 
         # Skip properties that we handle elsewhere
-        if prop.name in {"reg", "interrupts", "pwms", "clocks", "compatible"} or \
-           prop.name.endswith("gpios"):
+        if prop.name in {"reg", "interrupts", "compatible", "clocks"}:
```

Comment:
Shouldn't `clocks` also be defined with `type: compound` in `base.yaml`? It also has the format `phandle value phandle value ...` (except `#clock-cells` is usually 0).

Reply:
Fixed up base binding for clocks to be compound.